Add missing algs (#4809)
qgis/QGIS-Documentation: source/docs/user_manual/processing_algs/qgis/vectorgeometry.rst

     - The output (cleaned) vector layer

.. _qgisdeleteholes:

Delete holes
------------
Takes a polygon layer and removes holes in polygons.
It creates a new vector layer in which polygons with holes have been
replaced by polygons with only their external ring.
Attributes are not modified.

An optional minimum area parameter allows removing only holes which
are smaller than a specified area threshold. Leaving this parameter at
``0.0`` results in all holes being removed.

.. figure:: img/delete_holes.png
   :align: center

   Before and after the cleaning

|checkbox| Allows
:ref:`features in-place modification <processing_inplace_edit>`

Parameters
..........

.. list-table::
   :header-rows: 1
   :widths: 20 20 20 40
   :stub-columns: 0

   * - Label
     - Name
     - Type
     - Description
   * - **Input layer**
     - ``INPUT``
     - [vector: polygon]
     - Input polygon vector layer
   * - **Remove holes with area less than**

       Optional
     - ``MIN_AREA``
     - [number |dataDefined|]

       Default: 0.0
     - Only holes with an area less than this threshold will be
       deleted.

>        If ``0.0`` is added, **all** the holes will be deleted.

Reviewer: ```suggestion
       With a value of ``0.0``, **all** the holes will be deleted.
```
